### DISCUSSION TITLE
Not Able to Run RocketMQ on Intellij

### DISCUSSION THREAD

**Discussion by VishalMCF** [apache/rocketmq#7535] [Q&A]
Hello Team,
I was very hopeful that I will make contributions to the RocketMQ community but I am stuck at the beginning itself.
I cloned the project and when I load it into my IntelliJ i see errors on all the Apache maven plugins mentioned in the pom.xml of the project. Can someone please suggest something? I am really stuck and I have looked on the internet. 
I went through this articl but even this did not help -> https://stackoverflow.com/questions/20496239/maven-plugins-can-not-be-found-in-intellij
Later I loaded the project into the visual studio code and I was able to run this command successfully but not in IntelliJ-
mvn -Prelease-all -DskipTests -Dspotbugs.skip=true clean install -U

I ran this command after clearing my local .m2 repository
After running the command from Visual studio code I was able to run this command successfully -
nohup sh bin/mqnamesrv &
The process was successfully running on the port 9876.
But running this command to start broker and proxy -
nohup sh bin/mqbroker -n localhost:9876 --enable-proxy &

It showed errors in the nohup.out file

It maybe because in OpenJDK 20 UseBiasedLocking Option is not present

Has anyone faced this issue? I really need help as it is blocker for me to work on Apache eventmesh too. I am really hoping help from the community.

**Comment by Gezi-lzq** [comment]
Please check your Maven version and configuration, JDK version, and network status
